src/components/client.js

```diff
@@ -0,0 +1,9 @@
+import * as contentful from 'contentful';
+
+const client = contentful.createClient({
```

Comment:
There it is, amazing. I'll watch that video you linked about setting up a blog before filling this 9 lines of code with a billion questions, but let it be known that I understand very little of what's going on... some... but very little. That being said, what I do understand is that this is great. Lot's going on behind the scenes, and we're connected! Awesome work Rebecca!